packages/cli/templates/platform-app-tailwind/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+    "compilerOptions": {
+        "esModuleInterop": true,
+        "isolatedModules": true,
+        "module": "commonjs",
+        "moduleResolution": "node",
```

Comment:
Please use the new standards
https://www.totaltypescript.com/concepts/option-module-must-be-set-to-nodenext-when-option-moduleresolution-is-set-to-nodenext